lib/tbd/psi.rb

```diff
@@ -153,286 +153,301 @@ def initialize
-        joint:        0.376000, # placeholder for "cladding support"
-        transition:   0.000000  # defaults to 0
+        rimjoist:        0.307000, # "intermediate floor to wall intersection"
+        parapet:         0.260000, # "parapet" edge
+        roof:            0.020000, # (non-parapet) "roof" edge
+        fenestration:    0.194000, # "wall to vertical fenestration intersection"
+        door:            0.000000, # (unspecified, defaults to 0)
+        skylight:        0.000000, # (unspecified, defaults to 0)
+        spandrel:        0.000001, # (unspecified, defaults to 0)
+        corner:          0.000002, # (unspecified, defaults to 0)
+        balcony:         0.307000, # "intermediate floor balcony/overhang" edge
+        balconysill:     0.307000, # "intermediate floor balcony" edge (when sill)
+        balconydoorsill: 0.307000, # same as balcony
+        party:           0.000001, # (unspecified, defaults to 0)
+        grade:           0.000001, # (unspecified, defaults to 0)
+        joint:           0.376000, # placeholder for "cladding support"
+        transition:      0.000000  # defaults to 0
       }.freeze
 
       # ASHRAE 90.1 2022 (A10) "unmitigated" steel-framed and metal buildings.
       @set["90.1.22|steel.m|unmitigated"] =
       {
-        rimjoist:     0.842000, # "intermediate floor to wall intersection"
-        parapet:      0.500000, # "parapet" edge
-        roof:         0.650000, # (non-parapet) "roof" edge
-        fenestration: 0.505000, # "wall to vertical fenestration intersection"
-        door:         0.000000, # (unspecified, defaults to 0)
-        skylight:     0.000000, # (unspecified, defaults to 0)
-        spandrel:     0.000000, # (unspecified, defaults to 0)
-        corner:       0.000001, # (unspecified, defaults to 0)
-        balcony:      0.842000, # "intermediate floor balcony/overhang" edge
-        balconysill:  1.686000, # "intermediate floor balcony" edge (when sill)
-        party:        0.000001, # (unspecified, defaults to 0)
-        grade:        0.000001, # (unspecified, defaults to 0)
-        joint:        0.554000, # placeholder for "cladding support"
-        transition:   0.000000  # defaults to 0
+        rimjoist:        0.842000, # "intermediate floor to wall intersection"
+        parapet:         0.500000, # "parapet" edge
+        roof:            0.650000, # (non-parapet) "roof" edge
+        fenestration:    0.505000, # "wall to vertical fenestration intersection"
+        door:            0.000000, # (unspecified, defaults to 0)
+        skylight:        0.000000, # (unspecified, defaults to 0)
+        spandrel:        0.000001, # (unspecified, defaults to 0)
+        corner:          0.000002, # (unspecified, defaults to 0)
+        balcony:         0.842000, # "intermediate floor balcony/overhang" edge
+        balconysill:     1.686000, # "intermediate floor balcony" edge (when sill)
+        balconydoorsill: 0.842000, # same as balcony
```

Comment:
ASHRAE 90.1 2022 (A10) makes the distinction between:

 1. _intermediate floor/wall intersections_
 2. _intermediate floor/wall/balcony intersections_
 3. _intermediate floor/wall/balcony/"vertical fenestration"_ intersections

TBD inadvertently lumped _intermediate floor/wall/balcony/"opaque door"_ into the 3rd category. Thus a need for a 4th category:

4. _intermediate floor/wall/balcony/"opaque door"_ intersections (or simply **:balconydoorsill** edges)

And what PSI-factor to assign for _category 4_? TBD retains the most conductive PSI-factor that could apply, either:

- (opaque) door or doorsill
- balcony

As 90.1 2022 "_opaque door_" head/jamb/sill edges are set @0 W/K per linear meter (i.e. uncertainty on how to treat linear thermal bridges not explicitly listed in Table A10.1), **:balcony** PSI-factors apply.